### ISSUE TITLE
Should title be mandatory in ExposedThingInit?

### ISSUE THREAD

**Issue by relu91** [w3c/wot-scripting-api#300]
@mmccool pointed out that in the new ExposedThingInit algorithm title field should not be automatically generated. This field is critically important in UIs and human-readable documents. Therefore we should encourage developers to think twice before naming their Things with random unique strings. If they really need they can still their own function for the creation of unique names. This would also provide greater flexibility in name generation.

**Comment by danielpeintner** [comment]
I think we need to differ between `title` and `id`.

* `title` provides a human-readable title, see https://w3c.github.io/wot-thing-description/#thing. Hence no need to avoid collisions. I agree, the user script should provide it.
* `id` should be unique IMO. node-wot does behave like this, see  https://github.com/eclipse/thingweb.node-wot/blob/963a7537aa39de7d4af683b313bba96fe8ac1d6e/packages/core/src/servient.ts#L147

**Comment by relu91** [comment]
that's right I remembered a day when in node wot the title was used as id, but now it is fixed.
